server/app.js

```diff
@@ -1,18 +1,19 @@
 'use strict';
+global.__base = __dirname;

```

Comment:
When you use this a project this is './' - when you use hobknob to require it, it is going to be the main dir, not './node_modules/hobkbob'.

Basically, this is a pattern that you can do for when you run your node project, but not when you do something that is destined to be a dependency. We did that with OC too and then closed it.